src/NexusMods.App.UI/Pages/LoadoutPage/ICollectionLoadoutViewModel.cs

```diff
@@ -0,0 +1,31 @@
+using Avalonia.Media.Imaging;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.App.UI.WorkspaceSystem;
+using R3;
+
+namespace NexusMods.App.UI.Pages.LoadoutPage;
+
+public interface ICollectionLoadoutViewModel : IPageViewModelInterface
+{
+    LoadoutTreeDataGridAdapter Adapter { get; }
+
+    bool IsLocalCollection { get; }
+
+    bool IsReadOnly { get; }
```

Comment:
Could you add some comments on these? To explain what the definition is meant to be?